src/main/webapp/script.js

```diff
@@ -16,26 +16,61 @@ var clicked = false;
 
 function getVideoResults() {
   var i;
-  if (clicked == false) {
+  let directUrls = []
+  if (!clicked) {
       fetch('/videoResults').then(response => response.json()).then((data) => {
       const dataListElement = document.getElementById('videoResults');
       dataListElement.innerHTML = '';
       console.log(dataListElement.innerHTML);
       if (dataListElement.innerHTML == '') {
         for (i=0; i<data.length; i++) {
-          dataListElement.appendChild(
-            createIFrame(data[i]));
+          if (i % 2 == 0) {
```

Comment:
Is this logic to alternate between the embedded and direct URLs? Add a comment to describe this.

Reply:
I'll also do that